farm/file_utils.py

```diff
@@ -95,6 +96,36 @@ def filename_to_url(filename, cache_dir=None):
     return url, etag
 
 
+def download_from_s3(s3_url: str, cache_dir: str = "~/.cache/torch/farm"):
+    """
+    Download a "folder" from s3 to local. Skip already existing files. Useful for downloading all files of one model
+
+    :param s3_url: Url of the "folder" in s3 (e.g. s3://mybucket/my_modelname)
+    :param cache_dir: local directory where the files shall be stored
+    :return: local path of the folder
+    """
+
+    s3_resource = boto3.resource('s3')
+    bucket_name, s3_path = split_s3_path(s3_url)
+    bucket = s3_resource.Bucket(bucket_name)
+    objects = bucket.objects.filter(Prefix=s3_path)
+    logger.info(f"Downloading from {s3_url}")
+    for obj in objects:
```

Comment:
should we add a check here if objects exist?

Reply:
Added an exception